WzComparerR2/MainForm.cs

```diff
@@ -108,6 +109,78 @@ private void initFields()
+            UIStateConfig.Default.SplitterPosition1 = this.expandableSplitter1.SplitPosition;
+            UIStateConfig.Default.SplitterPosition2 = this.expandableSplitter2.SplitPosition;
+            UIStateConfig.Default.ColumnWidth3 = this.columnHeader3.Width.Absolute;
+            UIStateConfig.Default.ColumnWidth4 = this.columnHeader4.Width.Absolute;
+            UIStateConfig.Default.ColumnWidth5 = this.columnHeader5.Width.Absolute;
+            UIStateConfig.Default.ColumnWidth6 = this.columnHeader6.Width;
+            UIStateConfig.Default.ColumnWidth7 = this.columnHeader7.Width;
+            UIStateConfig.Default.ColumnWidth8 = this.columnHeader8.Width;
+            UIStateConfig.Default.ColumnWidth9 = this.columnHeader9.Width;
+            UIStateConfig.Default.BarLayout = this.dotNetBarManager1.LayoutDefinition;
+            ConfigManager.Save();
+        }
+
+        private void loadUIState()
+        {
+            try
+            {
+                this.WindowState = (FormWindowState)UIStateConfig.Default.WindowState.Value;
+                this.Size = new Size(UIStateConfig.Default.WindowWidth, UIStateConfig.Default.WindowHeight);
+                if(this.ribbonControl1.Expanded = UIStateConfig.Default.RibbonExpanded)
+                {
+                    switch(UIStateConfig.Default.SelectedRibbonTabIndex)
+                    {
+                        case 1: this.ribbonControl1.SelectedRibbonTabItem = this.ribbonTabItem1; break;
+                        case 2: this.ribbonControl1.SelectedRibbonTabItem = this.ribbonTabItem2; break;
+                        case 3: this.ribbonControl1.SelectedRibbonTabItem = this.ribbonTabItem3; break;
+                        default: this.ribbonControl1.SelectFirstVisibleRibbonTab(); break;
+                    }
+                }
+                this.expandableSplitter1.SplitPosition = UIStateConfig.Default.SplitterPosition1;
+                this.expandableSplitter2.SplitPosition = UIStateConfig.Default.SplitterPosition2;
+                this.columnHeader3.Width.Absolute = UIStateConfig.Default.ColumnWidth3;
+                this.columnHeader4.Width.Absolute = UIStateConfig.Default.ColumnWidth4;
+                this.columnHeader5.Width.Absolute = UIStateConfig.Default.ColumnWidth5;
+                this.columnHeader6.Width = UIStateConfig.Default.ColumnWidth6;
+                this.columnHeader7.Width = UIStateConfig.Default.ColumnWidth7;
+                this.columnHeader8.Width = UIStateConfig.Default.ColumnWidth8;
+                this.columnHeader9.Width = UIStateConfig.Default.ColumnWidth9;
+                this.dotNetBarManager1.LayoutDefinition = UIStateConfig.Default.BarLayout;
+            }
+            catch(Exception ex)
+            {
+                MessageBox.Show("Invalid ui data!");
+                this.WindowState = FormWindowState.Normal;
+                this.Size = new Size(1200, 800); // = new Size(766, 520);
```

Comment:
what will happen if screen size is small than that size?

Reply:
WcR2 will be bigger than the screen size.